apps/desktop-wallet/src/App.tsx

```diff
@@ -200,16 +198,8 @@ const App = () => {
     if (networkStatus === 'online') {
       if (addressesStatus === 'uninitialized') {
         if (!isSyncingAddressData && addressHashes.length > 0 && activeWalletId) {
-          const storedPendingTxs = pendingTransactionsStorage.load(activeWalletId)
-
           try {
             dispatch(syncAddressesData())
-              .unwrap()
-              .then((results) => {
-                const mempoolTxHashes = results.flatMap((result) => result.mempoolTransactions.map((tx) => tx.hash))
-
-                restorePendingTransactions(activeWalletId, mempoolTxHashes, storedPendingTxs)
-              })
```

Comment:
Now that we have 16s block time I don't see the need to store pending txs in local storage and restore them on app launch. I'd rather simplify and make less requests. This helps to debug the issue with the data sync.

Reply:
Definitely!